scripts/irods/lib.py

```diff
@@ -713,12 +713,57 @@ def get_replica_atime(session, logical_path, replica_number):
     out, _, _ = session.run_icommand(['iquery', query])
     return json.loads(out.strip())[0][0]
 
+
```

Comment:
not sure this is intentional

Reply:
It is intentional, though not necessary for these changes. When I removed the original implementation of `get_replica_full_row`, I left in the extra new line because PEP 8